src/ServicePulse.Host/app/js/directives/ui.particular.configurationTabs.tpl.html

```diff
@@ -1,7 +1,7 @@
 ﻿<div class="row">
     <div class="col-sm-12">
         <div class="tabs">
-            <h5 ng-class="{ active: isActive('/configuration/endpoints') }"><a href="#/configuration/endpoints">Endpoints ({{counters.endpoints | number}})</a></h5>
+            <h5 ng-class="{ active: isActive('/configuration/endpoints') }"><a href="#/configuration/endpoints">Endpoints monitoring ({{counters.endpoints | number}})</a></h5>
```

Comment:
`Endpoints monitoring` sounds strange, i feel like the tense is perhaps wrong.

What do you think about `Monitored endpoints`? @sergioc